src/test/groovy/unit/org/pih/warehouse/core/StockMovementSpec.groovy

```diff
@@ -0,0 +1,61 @@
+package unit.org.pih.warehouse.core
+
+import org.pih.warehouse.inventory.StockMovementStatusCode
+import org.pih.warehouse.requisition.Requisition
+import org.pih.warehouse.shipping.Shipment
+import org.pih.warehouse.shipping.ShipmentStatusCode
+import spock.lang.Specification
+import spock.lang.Unroll
+import org.pih.warehouse.requisition.RequisitionStatus
+import org.pih.warehouse.api.StockMovement
+
+@Unroll
+class StockMovementSpec extends Specification {
+
+    void 'StockMovement.getDisplayStatus() should return: #expected for requisition status #status'() {
+        when:
+        StockMovement stockMovement = new StockMovement() {
+            @Override
```

Comment:
Nice work on these tests!

In general I'm not the hugest fan of overriding methods in a test like this. Usually I'd try to Stub out the tagLib but it doesn't look like that'd be easy to do since it's fetched statically in the StockMovement. You could maybe Stub the StockMovemnt instance itself with something like:

```
StockMovement stockMovement = Stub(StockMovement) {
            getTranslatedDisplayStatus(def status) >> status.name()
        }
```
but that's essentially what you're doing anyways with your override (and I don't even know if the above code would work). I'm okay with your approach here.

Yay tests!